dna/zomes/projects/src/project/entry_point/validate.rs

```diff
@@ -5,6 +5,9 @@ use crate::project::{error::Error};
 use hdk::prelude::*;
 
 #[hdk_extern]
+/// A goal must exist at the entry point and the creator
+/// of the entry point must be the author of the goal unless
+/// the entry is imported
```

Comment:
Almost. The `creator_address` of the EntryPoint should match the agent authoring the EntryPoint. 

And you could say the `Goal` at the `goal_address` on the `EntryPoint` must exist, as opposed to "A goal must exist at the entry point"

Reply:
```suggestion
/// The `Goal` at the `goal_address` on the `EntryPoint` must exist and the `creator_address` of the
/// `EntryPoint` must match the agent authoring the `EntryPoint`, unless the entry is imported
```